build/FrameworkPackage/FrameworkPackageContents/AppxManifest.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10" xmlns:build="http://schemas.microsoft.com/developer/appx/2015/build" xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest" IgnorableNamespaces="uap build mp">
+  <Identity Name="$(PackageName)" ProcessorArchitecture="$(Platform)" Publisher="$(Publisher)" Version="$(Version)" />
+  <Properties>
+    <Framework>true</Framework>
+    <DisplayName>$(PackageName)</DisplayName>
+    <PublisherDisplayName>Microsoft Platform Extensions</PublisherDisplayName>
+    <Description>Microsoft Project Reunion</Description>
+    <Logo>logo.png</Logo>
+  </Properties>
+  <Resources>
+    <Resource Language="en-US" />
+  </Resources>
+  <Dependencies>
+    <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.15063.0" MaxVersionTested="10.0.18362.0" />
+  </Dependencies>
+  <mp:PhoneIdentity PhoneProductId="200b7352-04ae-4693-8a97-23a38e7d67d4" PhonePublisherId="1AC97219-40D1-4c98-B175-2BFCD96578C3" />
```

Comment:
Remove? (and the xml:mp namespace above and 'mp' in IgnorableNamespaces)

Mobile is no more (and hasn't been since well before any release we're targeting).

Store used to required PhoneIndentity for all submissions (though once Mobile was no more GUIDs of 00000000... were fine). Does Store still require this element? I hope not

Reply:
OK, have mail out on whether we want to coordinate any changes here with xaml (where all these files originated).

Reply:
mp removed

Reply:
I removed these, we can coordinate these changes in xaml later.